queries/landuse.jinja2

```diff
@@ -29,19 +52,27 @@ WHERE
 UNION ALL
 
 SELECT
-    name,
-    {{ ne_landuse_cols('park or protected land') }}
+  name,
+  tags->'protect_class' AS protect_class,
+  way_area::bigint AS area,
+  mz_calculate_landuse_kind("landuse", "leisure", "natural", "highway", "aeroway", "amenity", "tourism", "man_made", "power", "boundary") AS kind,
+  'openstreetmap.org' AS source,
+  {% filter geometry %}way{% endfilter %} AS __geometry__,
+  osm_id AS __id__
 FROM
-    ne_10m_parks_and_protected_lands
+  planet_osm_polygon
 WHERE
-    {{ bounds|bbox_filter('the_geom') }}
+  {{ bounds|bbox_filter('way') }}
+  AND boundary IN ('national_park', 'protected_area')
+  AND mz_one_pixel_zoom(way_area) <= {{ zoom }}
```

Comment:
We've discussed changing 1 pixel to something more configurable. The name should reflect that, maybe `mz_minimum_pixel_size_zoom(way_area)`? That's doesn't roll of the tongue, proposals?